toolkit/Makefile

```diff
@@ -155,7 +155,7 @@ VALIDATE_TOOLCHAIN_GPG ?= y
 endif
 endif
 
-TOOLCHAIN_GPG_VALIDATION_KEYS ?= $(wildcard $(SPECS_DIR)/azurelinux-repos/MICROSOFT-*-GPG-KEY) $(wildcard $(toolkit_root)/repos/MICROSOFT-*-GPG-KEY)
+TOOLCHAIN_GPG_VALIDATION_KEYS ?= $(wildcard $(PROJECT_ROOT)/SPECS/azurelinux-repos/MICROSOFT-*-GPG-KEY) $(wildcard $(toolkit_root)/repos/MICROSOFT-*-GPG-KEY)
```

Comment:
What does this do if you are a derivative building outside the tree?

Reply:
`$(wildcard ...)` will return the empty string if the directory doesn't exist, and the second call to ` $(wildcard $(toolkit_root)/repos/MICROSOFT-*-GPG-KEY)` should return the packaged copies.